docs/DeveloperGuide.md

```diff
@@ -452,23 +475,31 @@ Extensions:
 
   Use case ends.
 
-**Use case: UC03 - Find a student’s profile**
+**Use case: UC0X - Edit student details**
 
 MSS:
 
-1. User enters the find student command, along with a keyword from the student’s name.
-2. TutorBuddy displays all students’ profiles matching the keyword if any.
+1. User enters the `edit_student` command with the appropriate input.
+2. TutorBuddy verifies that the inputs are valid.
+3. TutorBuddy verifies that the student profile exists.
```

Comment:
What do you think about combining 2 and 3. I feel like they are the same since check whether the user exists is part of the input validation?
If so, then 2a and 3a below can also be combined to just 
2a. TutorBuddy detects and error in the input.
    2a1. TutorBuddy displays an error message to the user.
Use case ends.

Reply:
If you agree with this then maybe can standardise to other use cases as well also.

Reply:
Haha yeah, I originally did this, then I saw that our previous version had the 2 and 3 separated, so I went ahead with this. Since u agreed that combining is good also, I tink I will do that ah.

Reply:
Ok updated in the latest commit!